doc/src/explanation/active-deprecations.md

```diff
@@ -76,7 +76,125 @@ SymPy deprecation warnings.
+both bodies. And if a joint like the ``PinJoint`` has a specific joint axis,
+e.g. the axis around which the rotation occurs, than this axis can be specified
+using the ``joint_axis`` argument. A big advantage of this setup is that you can
+be more accurately define the transformation from parent body to child body.
+
+For example, suppose you want a ``PinJoint`` rotates the child body around the
+``parent.z`` axis and ``-child.z`` axis. The previous way to specify this joint
+was:
+
+```py
+>>> from sympy.physics.mechanics import Body, PinJoint
+>>> parent, child = Body('parent'), Body('child')
+>>> pin = PinJoint('pin', parent, child, parent_axis=parent.z,
+...                child_axis=-child.z)   # doctest: +SKIP
+>>> parent.dcm(child)   # doctest: +SKIP
+Matrix([
+[-cos(theta_pin(t)), -sin(theta_pin(t)),  0],
+[-sin(theta_pin(t)),  cos(theta_pin(t)),  0],
+[                 0,                  0, -1]])
+```
+
+When inspecting this matrix you will notice that for ``theta_pin = 0`` the child
+body is rotated $\pi$ rad around the ``parent.y`` axis. In the new definition
+you can see that we get the same result, but this time we have also specified
+this exact rotation:
+
+```py
+>>> from sympy import pi
+>>> from sympy.physics.mechanics import Body, PinJoint, ReferenceFrame
+>>> parent, child, = Body('parent'), Body('child')
+>>> int_frame = ReferenceFrame('int_frame')
+>>> int_frame.orient_axis(child.frame, child.y, pi)
+>>> pin = PinJoint('pin', parent, child, joint_axis=parent.z,
+...                child_interframe=int_frame)
+>>> parent.dcm(child)
+Matrix([
+[-cos(theta_pin(t)), -sin(theta_pin(t)),  0],
+[-sin(theta_pin(t)),  cos(theta_pin(t)),  0],
+[                 0,                  0, -1]])
+```
+
+However if you liked the fact that the deprecated arguments aligned the frames
+for you, than you can still make use of this feature by providing vectors to
+``parent_interframe`` and ``child_interframe``, which are than oriented such
+that the joint axis expressed in the intermediate frame is aligned with the
+given vector.
+
+```py
+>>> from sympy.physics.mechanics import Body, PinJoint
+>>> parent, child = Body('parent'), Body('child')
+>>> pin = PinJoint('pin', parent, child, parent_interframe=parent.z,
+...                child_interframe=-child.z)
+>>> parent.dcm(child)
+Matrix([
+[-cos(theta_pin(t)), -sin(theta_pin(t)),  0],
+[-sin(theta_pin(t)),  cos(theta_pin(t)),  0],
+[                 0,                  0, -1]])
+```
+
+(deprecated-mechanics-joint-pos)=
+### Change in joint attachment point argument
+
+The argument names for specifying the attachment points of a joint in
+``sympy.physics.mechanics`` , i.e. ``parent_joint_pos`` and ``child_joint_pos``,
+has been changed to ``parent_point`` and ``child_point``. Since these arguments
+can now also be ``Point`` objects, so they can be exactly the same as the
+``parent_point`` and ``child_point`` attributes.
```

Comment:
```suggestion
The argument names for specifying the attachment points of a joint in
``sympy.physics.mechanics`` , i.e. ``parent_joint_pos`` and ``child_joint_pos``,
have been changed to ``parent_point`` and ``child_point``. This is because these
arguments can now also be ``Point`` objects, so they can be exactly the same as
the ``parent_point`` and ``child_point`` attributes.
```